content/modules/redisearch/release-notes/redisearch-2.0-release-notes.md

```diff
@@ -0,0 +1,71 @@
+   - The RediSearch version in Redis will be 20000 or 2.0.0 in semantic versioning. Since the version of a module in Redis is numeric, we could not add an GA flag.
+   - RediSearch 2.x requires Redis 6.0 or later.
+
+### Behavior changes
+Please familiarize yourself with these changes before upgrading to RediSearch 2.0:
+  - [#1381](https://github.com/RediSearch/RediSearch/pull/1381): `FT.SYNADD` is removed; use `FT.SYNUPDATE` instead. Also the synonym group ID can now be a string (next to a number).
+  - [#1437](https://github.com/RediSearch/RediSearch/pull/1437): Documents that expire during query execution time will not appear in the results (but might have been counted in the number of produced documents).
+  - [#1221](https://github.com/RediSearch/RediSearch/pull/1221): Synonyms support for lower case. This can result in a different result set on FT.SEARCH when using synonyms.
+  - RediSearch will not index hashes whose fields do not match an existing index schema. You can see the number of hashes not indexed using [`FT.INFO`](https://oss.redislabs.com/redisearch/Commands/#ftinfo) - `hash_indexing_failures `.  The requirement for adding support for partially indexing and blocking is captured here: [#1455](https://github.com/RediSearch/RediSearch/pull/1455).
+  - Removed support for `NOSAVE` (for details see [v1.6 docs](https://oss.redislabs.com/redisearch/1.6/Commands/#ftadd)).
+  - RDB loading will take longer due to the index not being persisted.
+  - Field names in the [query syntax](https://oss.redislabs.com/redisearch/Query_Syntax/) are now case-sensitive.
+  - [Deprecated commands](https://oss.redislabs.com/redisearch/2.0/Commands/#deprecated_commands):
+    - `FT.DROP` (replaced by `FT.DROPINDEX`, which by default keeps the documents)
+    - `FT.ADD` (mapped to `HSET` for backward compatibility)
+    - `FT.DEL` (mapped to `DEL` for backward compatibility)
+    - `FT.GET` (mapped to `HGETALL` for backward compatibility)
+    - `FT.MGET` (mapped to `HGETALL` for backward compatibility)
+  - Removed commands:
+    - `FT.ADDHASH` (no longer makes sense)
+    - `FT.SYNADD` (see [#1381](https://github.com/RediSearch/RediSearch/pull/1381))
+    - `FT.OPTIMIZE` (see [v1.6 docs](https://oss.redislabs.com/redisearch/1.6/Commands/#ftoptimize))
+
+### Scaling changes
+In RediSearch 1.x, a single RediSearch index, and its documents, had to reside on a single shard. This meant that dataset size and throughput were limited to what a single Redis process could handle.
+
+Redis Enterprise now offers the ability to distribute documents in a clustered database and aggregate the results at query time. This fan-out and aggregation is handled by a component called the "coordinator", which is now also available under the same [Redis Source Available License](https://redislabs.com/legal/licenses/) for all Redis OSS users in its own repository: [RSCoordinator](https://github.com/RediSearch/RSCoordinator).
```

Comment:
@K-Jo This used to say "Redis Enterprise offered..." but that didn't make sense to me. Do you mean that Redis Enterprise now offers this?

Reply:
I mean that you can now do the same in Redis OSS and Redis Enterprise.  Prior it was only possible in Redis Enterprise

Reply:
Right so we removed support for synadd which normally returned an id for you.  you could supply this id on synadd.  While this might be cumbersome to create a unique sequence now, you can also use a string